### PR TITLE
Moving some slow tests to integration

### PR DESCRIPTION
**Pull request by andrewlee94**
## Summary/Motivation:
Run times for our non-integration tests have been creeping up again, so this PR moves a lot of slow tests (>10s) to integration. This will likely impact our apparent test coverage, as this is calculated based on the `unit` and `component` flags only, but cuts the run time for these by nearly 30% on my local machine.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: idaes/models/unit_models/tests/test_heat_exchanger_1D.py [hunk 1]
```diff
@@ -3544,7 +3544,7 @@ def model(self):
 
         return m
 
-    @pytest.mark.component
+    @pytest.mark.integration
     def test_general_hx1d_initializer(self, model):
         initializer = HX1DInitializer()
         initializer.initialize(model.fs.unit)
```
Comment (dallan-keylogic) [inline idaes/models/unit_models/tests/test_heat_exchanger_1D.py:3550]: I wonder how this initialization time is going to be affected when we move away from block triangularization in #1436 .

**Review by dallan-keylogic — APPROVED** [review]
LGTM

File: idaes/models/unit_models/tests/test_heat_exchanger_1D.py (hunk 1 above)
Comment (andrewlee94) [reply idaes/models/unit_models/tests/test_heat_exchanger_1D.py:3550]: That is a good point, but I suspect it won't have a huge effect in this case. I believe there is another equivalent test for a different configuration that runs fast enough, so I suspect this one is caused more by stiffness in the model than the property initialization. Still, would be worth waiting for #1436 to check.

Comment (andrewlee94) [reply idaes/models/unit_models/tests/test_heat_exchanger_1D.py:3553]: Update: after #1436 merged, run time for this test is still ~16 seconds, so I'll leave it in integration for now.

**Review by bpaul4 — APPROVED** [review]

**Review by bpaul4 — APPROVED** [review]

**Pull request merged**